eng/validation/update-channel.ps1

```diff
@@ -2,7 +2,7 @@ Param(
   [string] $maestroEndpoint,
   [string] $barToken,
   [string] $apiVersion = "2018-07-16",
-  [string] $targetChannelName = ".NET Eng - Latest",
+  [string] $targetChannelName = ".NET 6 Eng",
```

Comment:
Does it matter that darc get-channels shows a different casing (`".Net 6 Eng"`)?  Guessing not but figured I'd ask.

Reply:
I will update darc.. thanks for bringing this up :)

Reply:
Per Ricardo it doesn't actually matter and it's generally all caps, so I signed off.